Smarty/templates/BrowserVariables.tpl

```diff
@@ -26,6 +26,7 @@ var gcbpageisdirty = false;
 var gcbpagewhyisdirty = '';
 var goldcorebos_browsertabID = '{$corebos_browsertabID}';
 var gVTviewType = '{$gVTviewType}';
+var gVTsiteUrl = '{$gVTsiteUrl}';
```

Comment:
this should be a separate commit: try to make small and semantically cohesive commits. this one, with the setting below, is an individual feature that can be used in the whole application, it is not semantically related to this new feature even if it does use it